draft-ietf-gnap-core-protocol.md

```diff
@@ -1337,17 +1338,19 @@ uri (string)
 : User-facing web page of the client software
 
 logo_uri (string)
-: Display image to represent the client
-            software
+: Display image to represent the client software
 
 
 ~~~
 "display": {
     "name": "My Client Display Name",
-    "uri": "https://example.net/client"
+    "uri": "https://example.net/client", 
+    "logo_uri": "data:image/png;base64,Eeww...="
 }
 ~~~
 
+The logo MAY be passed by value by using a data: URI {{!RFC2397}} referencing an image mediatype.
```

Comment:
I suggest moving this sentence to the description of "logo_uri", I don't think you want someone to think it's allowed for other places like "uri" and so forth.

Reply:
yes we can do that